### PR TITLE
Upgrade ol5

### PR DESCRIPTION
**Pull request by dnlkoch**
This updates the `ol-util` classes to make use of the current version of `ol` (5.1.3).

In addition eslint is added.

### CODE REVIEW

File: src/MeasureUtil/MeasureUtil.spec.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ describe('MeasureUtil', () => {
         const expectedShortLength = MeasureUtil.getLength(shortLine, map);
         const expectedLongLength = MeasureUtil.getLength(longLine, map);
 
-        expect(expectedShortLength).toEqual(99.99999999669033);
-        expect(expectedLongLength).toEqual(100545.83533277796);
+        expect(expectedShortLength).toEqual(99.88824008937313);
```
Comment (ahennr) [inline src/MeasureUtil/MeasureUtil.spec.js]: Why did these values change?

Comment (hwbllmnn) [reply src/MeasureUtil/MeasureUtil.spec.js]: Also, floats/doubles should not be checked for equality, but compared using a delta (`0.1 + 0.2 !== 0.3).

Comment (dnlkoch) [reply src/MeasureUtil/MeasureUtil.spec.js]: The default sphere radius taken into account for length/area calculation in `ol` is set to [6371008.8](https://github.com/openlayers/openlayers/blob/v5.1.3/src/ol/sphere.js#L32) whereas the previous implementation in `ol-util` used the non-default value [6378137](https://github.com/terrestris/ol-util/blob/master/src/MeasureUtil/MeasureUtil.js#L24) (that is the value equal to the semi-major axis of the WGS84 ellipsoid).

If needed the value may be passed to the corresponding util methods, but in here I would like to make use of the same radius as used in ol per default.

Comment (dnlkoch) [reply src/MeasureUtil/MeasureUtil.spec.js]: @hwbllmnn Yes, absolutely! Fixed (and will be fixed for all other occurrences in a separate PR).

**Pull request merged**